modules/rosa-delete-objects.adoc

```diff
@@ -0,0 +1,278 @@
+|===
+
+
+.Optional arguments inherited from parent commands
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--help
+|Shows help for this command.
+
+|--debug
+|Enables debug mode.
+
+|--interactive
+|Enables interactive mode.
+
+|--profile
+|Uses a specific AWS profile (string) from your credentials file.
+
+|--v level
+|Log level for V logs.
+|===
+
+.Example
+Delete a cluster administrator from a cluster named `mycluster`.
+
+[source,terminal]
+----
+$ rosa delete admin --cluster=mycluster
+----
+
+[id="rosa-delete-cluster_{context}"]
+== delete cluster
+
+Deletes a cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa delete cluster --cluster=<id> | <name> [arguments]
+----
+
+.Arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--cluster
+|(Required) The name or ID (string) of the cluster to delete.
+
+|--watch
+|Watches the cluster uninstallation logs.
+|===
+
+.Optional arguments inherited from parent commands
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--help
+|Shows help for this command.
+
+|--debug
+|Enables debug mode.
+
+|--interactive
+|Enables interactive mode.
+
+|--profile
+|Uses a specific AWS profile (string) from your credentials file.
+
+|--v level
+|Log level for V logs.
+
+|--yes
+|Automatically answers yes to confirm the operation.
+|===
+
+.Examples
+Delete a cluster named `mycluster`.
+
+[source,terminal]
+----
+$ rosa delete cluster --cluster=mycluster
+----
+
+[id="rosa-delete-idp_{context}"]
+== delete idp
+
+Deletes a specific identity provider (IDP) from a cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa delete idp <id> | <name> [arguments]
+----
+
+.Arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--cluster
+|(Required) The name or ID (string) of the cluster from which the IDP will be deleted.
+|===
+
+.Optional arguments inherited from parent commands
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--help
+|Shows help for this command.
+
+|--debug
+|Enables debug mode.
+
+|--interactive
+|Enables interactive mode.
+
+|--profile
+|Uses a specific AWS profile (string) from your credentials file.
+
+|--v level
+|Log level for V logs.
+
+|--yes
+|Automatically answers yes to confirm the operation.
+|===
+
+.Example
+Delete an identity provider named `github-1` from a cluster named `mycluster`.
+
+[source,terminal]
+----
+$ rosa delete idp github-1 --cluster=mycluster
+----
+
+[id="rosa-delete-ingress_{context}"]
+== delete ingress
+
+Deletes a non-default application router (ingress) from a cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa delete ingress [arguments]
+----
+
+.Arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--cluster
+|(Required) The name or ID (string) of the cluster from which the ingress will be deleted.
+|===
+
+.Optional arguments inherited from parent commands
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--help
+|Shows help for this command.
+
+|--debug
+|Enable debug mode.
+
+|-i, --interactive
```

Comment:
Tiny nit: in some parts of the doc set, you list the long-form flag only (`--interactive`)

Reply:
Fixed. Using long-form only.